cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/Changelog.java

```diff
@@ -80,11 +81,13 @@ public void onVariantCreated(String sourceVariantId, String targetVariantId) {
 
     public List<IidmChange> getChangesForVariant(String variantId) {
         if (!changesByVariant.containsKey(variantId)) {
+            // If we only have baseChanges we assume they are already ordered
             return Collections.unmodifiableList(baseChanges);
         } else {
             List<IidmChange> cs = new ArrayList<>(baseChanges.size() + changesByVariant.size());
             cs.addAll(baseChanges);
             cs.addAll(changesByVariant.get(variantId));
+            cs.sort(Comparator.comparing(IidmChange::getIndex));
```

Comment:
If the order is important, maybe we should use a TreeSet, with a comparator on the index?

Reply:
Done